src/Contracts/Driver.php

```diff
@@ -42,5 +48,7 @@ public function message(string $message): Driver
         return $this;
     }
 
+    abstract protected function boot(): void;
```

Comment:
Not every driver will need a boot method. It might be better to not make it `abstract`.

Reply:
🤔  yes. it's a possibility. Not forcing devs to implement it is a good option by the way.